data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
 		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
 
+
+mission "Derelict Informer"
+	minor
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "uninhabited"
+	to offer
+		random < 4
+		"days since start" > 30
+	on offer
+		conversation
+			`As you touch down on <origin>, you see that the landing pad next to yours holds a freighter that is practically falling apart. Standing next to it is a man with the grizzled look of an experienced captain.`
+			`	Given the sorry state of his ship he might be in need of assistance.`
+			choice
+				`	(Offer your help.)`
+				`	(Don't help him.)`
+					decline
+			`	The captain notices you approaching, and clearly guessing your intentions, speaks. "Oh don't worry, this ain't my ship."`
+			choice
+				`	"What are you doing with it then?"`
+					goto what
+				`	"Whose is it then?"`
+			`	"Well quite simply, it's nobody's."`
+			choice
+				`	"What are you doing with it then?"`
+				`	"If it isn't yours, why do you have it?"`
+			label what
+			`	An easy, broad smile upon his face tells you he's heard such a question before. "Glad you asked. It's always a pleasure to teach another captain about the niche trade of derelict hunting."`
+			choice
+				`	"Derelict hunting?"`
+					goto confused
+				`	"So you search for abandoned ships?"`
+					goto precisely
+			label confused
+			`	"Derelict hunting involves searching abandoned ship hulls. They usually have a few parts left on them that you can sell for a few credits, and if you're lucky you might even score some rare outfits or the whole hull if it's stable enough. Wouldn't recommend actually using that hull though. They're usually very fragile like this freighter here."`
```

Comment:
All of the derelict variants are uncapturable. Not sure how much it makes sense to say that you can capture derelict ships.